### PR TITLE
XW-3171 | banner notifications

### PR DESCRIPTION
**Pull request by latata**
https://wikia-inc.atlassian.net/browse/XW-3171
@JamesHawking @rybmat

### CODE REVIEW

File: app/templates/route-components/banner-notifications.hbs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+		</div>
+	{{/component-demo}}
+</section>
+
+<section>
+	<h2 id="text-only">Warning</h2>
+	<p>
+		Add <code class="format-code">.wds-warning</code> class to <code class="format-code">
+		.wds-banner-notification</code> element and use icon <code
+		class="format-code">wds-icons-alert2-small</code>
```
Comment (rybmat) [inline app/templates/route-components/banner-notifications.hbs]: alert2?

File: app/templates/route-components/banner-notifications.hbs [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+		</div>
+	{{/component-demo}}
+</section>
+
+<section>
+	<h2 id="text-only">Success</h2>
+	<p>
+		Add <code class="format-code">.wds-success</code> class to <code class="format-code">
+		.wds-banner-notification</code> element and use icon <code
+		class="format-code">wds-icons-checkmark-circle2-small</code>
```
Comment (rybmat) [inline app/templates/route-components/banner-notifications.hbs]: circle2?

File: app/templates/route-components/banner-notifications.hbs [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+			{{#wds-banner-notification type='message'}}
+				This is a general notification message.
+			{{/wds-banner-notification}}
+		</div>
+	{{/component-demo}}
+</section>
+
+
+<section>
+	<h2 id="text-only">Stacked</h2>
```
Comment (hakubo) [inline app/templates/route-components/banner-notifications.hbs]: should that be class?

Comment (latata) [reply app/templates/route-components/banner-notifications.hbs]: There should be no attributes.

File: style-guide/styles/wds-components/_banner-notification.scss [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+		padding: 13px 12px;
+	}
+
+	&__close {
+		cursor: pointer;
+		padding: 18px;
+	}
+
+	+ .wds-banner-notification .wds-banner-notification {
+		&__close, &__text {
```
Comment (vforge) [inline style-guide/styles/wds-components/_banner-notification.scss]: This nesting is not needed - Also the whole rule can be shortened into:
```scss
& + & &__close,
& + & &__text {
```
Which will be more explicit

File: style-guide/styles/wds-components/_banner-notification.scss [hunk 5]
```diff
@@ -0,0 +1,52 @@
+.wds-banner-notification-container {
```
Comment (vforge) [inline style-guide/styles/wds-components/_banner-notification.scss]: This is not a separate component - it's tied to the `banner-notification` and `banner-notification` cannot be used without the container. I suggest at leat renaming it to `banner-notification__container`.

**Pull request merged**